workers/csharp_physics/src/Main.cs

```diff
@@ -89,15 +89,16 @@ private static int Main(string[] args)
                     angle += 0.5f;
                     Position.Update positionUpdate = new Position.Update();
                     positionUpdate.coords.Set(new Coordinates(Math.Sin(angle) * 10, 0.0, Math.Cos(angle) * 10));
-                    connection.SendComponentUpdate(ourEntity, positionUpdate);
+                    connection.SendComponentUpdate(Position.Metaclass, ourEntity, positionUpdate);
 
                     // Sleep for some time.
                     System.Threading.Thread.Sleep(1000);
 
                     // Send a command every 2 ticks.
                     if (tickCount % 2 == 0)
                     {
-                        connection.SendCommandRequest(ourEntity, new ClientData.Commands.TestCommand.Request(10, 1.5f), null);
+                        connection.SendCommandRequest(ClientData.Commands.TestCommand.Metaclass, 
```

Comment:
Hmm. I think the upgrade guide I wrote for changing to the new C# syntax might not be 100% correct when changing the request parameter (I wrote something like `new TestCommand.Request(new Sample.AddCommandRequest(...))`, when I should've written `new TestCommand.Request(...)` cause the old wrapper type has a constructor that forwards the constructor arguments). Maybe it's fine, and I can just fix it when back on the 22nd.

Reply:
You wrote
```cs
// before
var request = new MyRequest(123, 89.0);
connection.SendCommandRequest<MyComponent.Commands.TestCommand>(entityId, new MyComponent.Commands.TestCommand.Request(request), ...);
// after
var request = new MyRequest(123, 89.0);
connection.SendCommandRequest(MyComponent.Commands.TestCommand.Metaclass, entityId, request, ...);
```
which I think is correct?